test/conf/creds.conf.sample

```diff
@@ -3,10 +3,10 @@ host = "dev-test-1"
 [psql]
     database = "db_test"
     port = 5432
-    user = "user"
-    password = "password"
+    user = "postgres"
+    password = "366369"
```

Comment:
Pls revert the sample file to the original

Reply:
I had done it

Reply:
I had done it